src/test/java/com/google/api/generator/engine/ast/JavaDocCommentTest.java

```diff
@@ -62,9 +62,42 @@ public void createJavaDocComment_sampleCode() {
         JavaDocComment.builder().addComment(comment).addSampleCode(sampleCode).build();
     String expected =
         "sample codes:\n"
-            + "<pre><code>\n"
+            + "<pre>{@code\n"
             + "resource = project/{project}/shelfId/{shelfId}\n"
-            + "</code></pre>";
+            + "}</pre>";
+    assertEquals(javaDocComment.comment(), expected);
+  }
+
+  @Test
+  public void createJavaDocComment_sampleCodeKeepIndent() {
+    String comment = "sample codes:";
+    String sampleCode = "try (condition = false) {\n" + "  int x = 3;\n" + "}";
+    JavaDocComment javaDocComment =
+        JavaDocComment.builder().addComment(comment).addSampleCode(sampleCode).build();
+    String expected =
+        "sample codes:\n"
+            + "<pre>{@code\n"
+            + "try (condition = false) {\n"
+            + "  int x = 3;\n"
+            + "}\n"
+            + "}</pre>";
+    assertEquals(javaDocComment.comment(), expected);
```

Comment:
Could you please reverse the `expected` and `javaDocComment.comment()` arguments? This is because `assertEquals` expects the `expected` result as the first argument, so the test logs are printed as such.

Same throughout.